src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -102,28 +102,6 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 				)
 			}
 			.store(in: &subscriptions)
-		
-		viewModel.cclService.shouldShowNoticeTile
```

Comment:
With this deletion we would ignore the CCL config for App Closure Notice. I know it makes sense, because we show either the notice or the thank-you tile. But is it aligned with the Tech Spec?